styleguide/src/styleguide/components/SectionPage.vue

```diff
@@ -61,7 +61,7 @@ export default {
     }
 
     try {
-      const mdFile = require(`!raw-loader?modules!../docs/${name}.md`)
+      const mdFile = require(`raw-loader!../docs/${name}.md`)
```

Comment:
:thinking:

Reply:
Yeah, they changed the syntax for the raw loader